stl/inc/yvals.h

```diff
@@ -178,6 +178,14 @@ _STL_DISABLE_CLANG_WARNINGS
 #error _ITERATOR_DEBUG_LEVEL != 0 must imply _CONTAINER_DEBUG_LEVEL == 1.
 #endif // _ITERATOR_DEBUG_LEVEL != 0 && _CONTAINER_DEBUG_LEVEL == 0
 
+#if _CONTAINER_DEBUG_LEVEL > 0
+#define _TEMPLATE_CDL  template <int _Cdl = _CONTAINER_DEBUG_LEVEL>
+#define _COMMA_INT_CDL , int _Cdl = _CONTAINER_DEBUG_LEVEL

```

Comment:
I suppose I should phrase this the other way around: these shouldn't be named since we don't intend to refer to these template parameters; having names could result in shadowing warnings, but produces no positive effect I can see.